kafkakewl-deploy/src/main/scala/com/mwam/kafkakewl/deploy/endpoints/Endpoints.scala

```diff
@@ -6,17 +6,29 @@
 
 package com.mwam.kafkakewl.deploy.endpoints
 
-import sttp.tapir.PublicEndpoint
+import com.mwam.kafkakewl.common.http.{EndpointUtils, ErrorResponse}
+//import com.mwam.kafkakewl.deploy.domain.{Failures, QueryFailure}
+import sttp.model.StatusCode
+import sttp.tapir.{EndpointOutput, PublicEndpoint}
 import sttp.tapir.swagger.bundle.SwaggerInterpreter
 import sttp.tapir.ztapir.*
+//import sttp.tapir.ztapir.a
 import zio.*
+import sttp.tapir.generic.auto.*
+import sttp.tapir.json.zio.jsonBody
 import zio.metrics.connectors.prometheus.PrometheusPublisher
 
+import zio.json.{DeriveJsonDecoder, DeriveJsonEncoder, JsonDecoder, JsonEncoder}
+
 class Endpoints(
     deploymentServerEndpoints: DeploymentsServerEndpoints,
     prometheusPublisher: PrometheusPublisher
 ) {
-  private val metricsEndpoint: PublicEndpoint[Unit, Unit, String, Any] = endpoint.in("metrics").get.out(stringBody)
+  private val metricsEndpoint: PublicEndpoint[Unit, Unit, String, Any] =
```

Comment:
nitpick: did scalafmt split this line up or it was done manually? (It's only 114 characters long)

Reply:
Ah yes if you make a line longer (I used metrics to test out the timeout), then scalafmt will split it up, but if you undo it, scalafmt won't put them back onto one line. I'll find a way to make scalafmt do this